src/MSALWrapper/PCAWrapper.cs

```diff
@@ -93,19 +109,29 @@ public interface IPCAWrapper
     /// </summary>
     public class PCAWrapper : IPCAWrapper
     {
+        private ILogger logger;
         private IPublicClientApplication pca;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="PCAWrapper"/> class.
         /// </summary>
-        /// <param name="pca">
-        /// The pca.
-        /// </param>
+        /// <param name="pca">The public client application instance.</param>
         public PCAWrapper(IPublicClientApplication pca)
         {
             this.pca = pca;
         }
 
+        /// <summary>

```

Comment:
(I don't expect it to be that difficult to pass in a logger in the existing uses).

Reply:
Are you suggesting the following:
`        public PCAWrapper(ILogger logger = null, IPublicClientApplication pca)
        {
            this.logger = logger;
            this.pca = pca;
        }
`

Just have on constructor like this and fix `TokenFetcherPublicClient` and other existing classes to take a logger in the `PCAWrapper` constrictor? Please elaborate so that I can fully understand how you want these. :)

Reply:
Almost - just drop the `= null`. So 
```chsarp
public PCAWrapper(ILogger logger, IPublicClientApplication pca)
{
  this.logger = logger;
  this.pca = pca;
}
```

Reply:
Done!